src/command.ts

```diff
@@ -61,6 +61,10 @@ export async function runCommand<T extends ArgsDef = ArgsDef>(
   if (typeof cmd.run === "function") {
     await cmd.run(context);
```

Comment:
We need to keep error here and rethrow it after cleanup to guarantee this hook get's called

Reply:
Oh yeah, I see the point. If an error is thrown, this hook will never be called.